index.d.ts

```diff
@@ -4,4 +4,7 @@ declare global {
   interface Window {
     CustomElementElement: typeof CustomElementElement
   }
-}
+  interface HTMLElementTagNameMap {
+    'custom-boilerplate': CustomElementBoilerplate
```

Comment:
😬 😆 This one should be `'custom-element': CustomElementElement`

Reply:
Lol, my script was very hacky 😄

Reply:
```suggestion
    'custom-element': CustomElementElement
```